src/components/organizationItem.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const organizationItem = props => {
+  return (
+    <a className="sponsor-url" href={props.item.url}  target="_blank" name={props.item.name}>
```

Comment:
Creo que acá podrías tener algo como: 
`const { item: { image, url, name } } = this.props;`

Reply:
Siempre he sido partidario de definir los props por PropTypes pero podemos definir una regla para todos los componentes.